builtin/credential/cert/path_certs.go

```diff
@@ -228,6 +234,18 @@ func (b *backend) pathCertWrite(ctx context.Context, req *logical.Request, d *fr
 		}
 	}
 
+	var parsedCIDRs []*sockaddr.SockAddrMarshaler
+	for _, v := range d.Get("bound_cidrs").([]string) {
+		parsedCIDR, err := sockaddr.NewSockAddr(v)
+		if err != nil {
+			if b.Logger().IsDebug() {
+				b.Logger().Debug(fmt.Sprintf("unable to parse %s as a cidr: %s", v, err))
+			}
+			return nil, logical.ErrPermissionDenied
```

Comment:
Rather than a permission denied error here we should return an `ErrInvalidRequest`, probably with an error response with the parse error

Reply:
:heavy_check_mark: